app-i18n/mozc/mozc-2.20.2677.102.recipe

```diff
@@ -0,0 +1,97 @@
+SUMMARY="Japanese input method editor"
+DESCRIPTION="A Japanese Input Method Editor (IME) designed for multi-platform. \
+This is open source version."
+HOMEPAGE="https://github.com/google/mozc/"
+COPYRIGHT="2010-2016, Google Inc."
+LICENSE="BSD (3-clause)"
+REVISION="1"
+fileVersion="v2.20.2677.102.3"
+SOURCE_URI="https://github.com/hanya/mozc/releases/download/$fileVersion/mozc-$fileVersion.tar.bz2"
+CHECKSUM_SHA256="5b7c8c1e114b99221a2d9cf650fadd1cad9137986c94762f490a62af1048a8b1"
+SOURCE_DIR="mozc"
+if [ $effectiveTargetArchitecture == "x86" ]; then
+    PATCHES="mozc-$portVersion-x86.patch"
+fi
+SOURCE_URI_2="https://jaist.dl.sourceforge.net/project/zinnia/zinnia-tomoe/0.6.0-20080911/zinnia-tomoe-0.6.0-20080911.tar.bz2"
+CHECKSUM_SHA256_2="fd1a264b8aeea0ebec59e8c8765864a9543a75fb2dba349745bb3a3587790927"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+
+PROVIDES="
+    mozc
+    addon:mozc
+    "
+REQUIRES="
+    haiku
+    lib:libQt5Core
+    lib:libQt5Gui
+    lib:libQt5Widgets
+    "
+BUILD_REQUIRES="
+    haiku_devel
+    cmd:python
+    cmd:ninja
+    protobuf
+    cmd:protoc
+    protobuf_devel
+    lib:libprotobuf
```

Comment:
You shouldn't need this; protobuf_devel should pull it in automatically. (And replacing `protobuf_devel` with `devel:libprotobuf` would be preferred, too.)